### PR TITLE
tool: misc cleanup of ceph-kvstore-tool

### PR DESCRIPTION
**Pull request by Liuchang0812**
1. fixes `store-crc <path>` command. this command did not do anything before.
2. uses coarse monotonic clock

@tchaikov @liewegas would you mind taking a look? thanks

### CODE REVIEW

**Comment by Liuchang0812** [comment]
jenkins test this please

File: src/tools/ceph_kvstore_tool.cc [hunk 1]
```diff
@@ -558,8 +559,14 @@ int main(int argc, const char *argv[])
...
   } else if (cmd == "store-crc") {
-    uint32_t crc = st.traverse(string(), true, NULL);
-    std::cout << "store at '" << path << "' crc " << crc << std::endl;
+    if (argc < 4) {
+      usage(argv[0]);
+      return 1;
+    }
+    std::ofstream fs(argv[4]);
+    uint32_t crc = st.traverse(string(), true, &fs);
+    fs.close();
```
Comment (tchaikov) [inline src/tools/ceph_kvstore_tool.cc]: "fix" is too vague for me. could you expand a little bit? also, `fs` will be closed when it's restructured. i don't think this call is really necessary.

Comment (Liuchang0812) [reply src/tools/ceph_kvstore_tool.cc]: thank you, kefu, updated!

File: src/tools/ceph_kvstore_tool.cc [hunk 2]
```diff
@@ -558,8 +558,13 @@ int main(int argc, const char *argv[])
     }
 
   } else if (cmd == "store-crc") {
-    uint32_t crc = st.traverse(string(), true, NULL);
-    std::cout << "store at '" << path << "' crc " << crc << std::endl;
+    if (argc < 4) {
+      usage(argv[0]);
+      return 1;
+    }
+    std::ofstream fs(argv[4]);
```
Comment (tchaikov) [inline src/tools/ceph_kvstore_tool.cc]: it should be `argv[3]`. as cmd is `argv[2]`.

Comment (Liuchang0812) [reply src/tools/ceph_kvstore_tool.cc]: it should be argv[4].  

```
➜  build git:(wip-kv-store-crc) ✗ ./bin/ceph-kvstore-tool bluestore-kv dev/osd2 store-crc t2.txt 2>/dev/null 
store at 't2.txt' crc 4037852868
```

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-11-10_06:12:07-rados-wip-kefu-testing-2017-11-10-1154-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2017-11-12_17:42:28-rados-wip-kefu-testing-2017-11-10-1154-distro-basic-mira/

**Pull request merged**